packages/components/src/components/button/_button.scss

```diff
@@ -22,7 +23,7 @@
   }
 
   .#{$prefix}--btn-set > .#{$prefix}--btn {
-    max-width: rem(196px); // taken from design kit
+    max-width: rem(199px); // 196px from design kit, 3px for new border width
```

Comment:
Any background wrt button border vs. outline width topic? (My apologies if I'm missing something in what you brought up earlier)

Reply:
thanks for spotting this, I believe this was changed temporarily and I did not revert the change